db/seeds/002_recipes.rb

```diff
@@ -168,11 +181,18 @@
     notes: "Add grilled chicken for a protein boost.",
     rating: 4.2,
     is_favorite: false,
+    categories: %w[Salad Appetizer Side]
   },
 ]
 
 recipes_data.each do |recipe_data|
-  Recipe.find_or_initialize_by(name: recipe_data[:name]).update(recipe_data)
+  recipe = Recipe.find_or_initialize_by(name: recipe_data[:name])
+  categories = recipe_data.delete(:categories) { [] }
+  recipe.category_ids = Category.where(name: categories).pluck(:id)
```

Comment:
We might want to create the categories if they don't exist.  The suggested `from_names` method on the `Category` model is useful here.
```suggestion
  category_names = recipe_data.delete(:categories) { [] }
  recipe.categories = Category.from_names(category_names)
```

Reply:
Actually, this can be simplified now with category_names